api_tests/tests/ether_halight.ts

```diff
@@ -10,13 +10,14 @@ import { sleep } from "../src/utils";
 it(
     "han-ethereum-ether-halight-lightning-bitcoin-alice-redeems-bob-redeems",
     twoActorTest(async ({ alice, bob }) => {
-        const [aliceBody, bobBody] = await SwapFactory.newSwap(alice, bob);
+        const bodies = (await SwapFactory.newSwap(alice, bob))
+            .hanEthereumEtherHalightLightningBitcoin;
```

Comment:
I've recently found that using a mix of `then` and await is a nice way of chaining things :)


```suggestion
        const bodies = await SwapFactory.newSwap(alice, bob).then(bodies => bodies.hanEthereumEtherHalightLightningBitcoin)
```